### ISSUE TITLE
place_seqs.py create tre file, but out.tre generates no files from hsp.py

### ISSUE THREAD

**Issue by jtclaypool** [picrust/picrust2#183]
Working on a dataset that runs fine without the place_seqs.py flag "-t sepp", but when using sepp, a tree file is produced but does not work in the subsequent hsp.py steps. 

I am passing a seqs.fna file and a hd5.biom file to an Apache Airflow environment running a docker container (https://quay.io/repository/biocontainers/picrust2) running picrust2. 

`place_seqs.py -s ./sequences.fna -o ./$outdir/out.tre -p 16 --intermediate intermediate/place_seqs -t sepp;`

will produce out.tre, but when fed into hsp.py, I get no output or error from the system.
For example
`hsp.py -i 16S -t $outdir/out.tre -o $outdir/marker_predicted_and_nsti.tsv.gz -p 16 -n;`

produces no file or error message. 

I am not sure where the error is occurring or where the mishap is. It is fixed by removing "-t sepp" from place_seqs.py, but I don't understand why I would have an out.tre generated but have it not work nor receive any error. I'm assuming I haven't given enough RAM to the instance on Airflow but wanted to pass this along in case anybody else has a similar issue. Fixed for our work, but let me know if I can help diagnose the issue. Data is confidential, but I could look into synthetically generating data to reproduce the error if desired. Could also be the docker container. Or Airflow. 

To sum it up. Fixed by removing "-t sepp".

**Comment by gavinmdouglas** [comment]
Thanks for posting this issue and solution @jtclaypool!
